web/controllers/organization_membership_controller.ex

```diff
@@ -0,0 +1,82 @@
+defmodule CodeCorps.OrganizationMembershipController do
+  use CodeCorps.Web, :controller
+
+  import CodeCorps.ControllerHelpers
+
+  alias JaSerializer.Params
+
+  alias CodeCorps.OrganizationMembership
```

Comment:
Based on https://github.com/levionessa/elixir_style_guide#modules, which, according to stack overflow is the most commonly used style guide, it's

```
@moduledoc
use
import
alias
require
@type
@module_attribute
```